models/user.js

```diff
@@ -0,0 +1,30 @@
+const mongoose = require('mongoose');
+
+const Schema = mongoose.Schema; // Used to create a new Schema
+
+// Define the Schema for user
+// TODO we can check for validation in the future
+const userSchema = new Schema({
+    firstName: {
+        type: String,
+        required: true
+    },
+    lastName: {
+        type: String,
+        required: true
+    },
+    gender: {
+        type: String,
+        required: true
+    },
+    age: {
+        type: Number,
+        required: true
+    },
+    email: {
+        type: String,
+        required: true
+    }
```

Comment:
This is a good start! We will add more schema in the future in terms of the people they like to be room with!